colossus-tests/src/it/scala/colossus/BaseRedisITSpec.scala

```diff
@@ -3,20 +3,27 @@ package colossus
 import java.net.InetSocketAddress
 
 import akka.util.ByteString
-import colossus.metrics.MetricSystem
+import colossus.metrics.{MetricReporterConfig, MetricSystem}
 import colossus.protocols.redis._
 import colossus.service.ClientConfig
 import colossus.testkit.ColossusSpec
-import org.scalatest.concurrent.{ScaledTimeSpans, ScalaFutures}
+import org.scalatest.concurrent.{ScalaFutures, ScaledTimeSpans}
 import org.scalatest.time.Span
 import org.scalatest.time._
 
 import scala.concurrent.Future
 import scala.concurrent.duration._
 
 abstract class BaseRedisITSpec extends ColossusSpec with ScalaFutures with ScaledTimeSpans {
+  val metricSystem =  MetricSystem("test-system")
 
-  implicit val sys = IOSystem("test-system", Some(2), MetricSystem.deadSystem)
+  val reporterConfig = MetricReporterConfig(
```

Comment:
You don't need to create `reporterConfig` or set it up since you are not using the reporter for anything. Just need to create a `MetricSystem`